packages/hooks/src/useExistingCFDAccounts.ts

```diff
@@ -0,0 +1,71 @@
+import { useEffect, useMemo } from 'react';
+import { useRequest } from '@deriv/api';
+import useActiveWallet from './useActiveWallet';
+import { useStore } from '@deriv/stores';
+
+/**
+ * @description This hook is used to get the created CFD accounts of the user.
+ */
+const useExistingCFDAccounts = () => {
+    const { traders_hub } = useStore();
+    const { combined_cfd_mt5_accounts } = traders_hub;
+    const wallet = useActiveWallet();
+    const { mutate: mt5_mutate, ...mt5 } = useRequest('mt5_login_list');
+    const { mutate: derivez_mutate, ...derivez } = useRequest('trading_platform_accounts');
+    const { mutate: dxtrade_mutate, ...dxtrade } = useRequest('trading_platform_accounts');
+
+    useEffect(() => {
+        mt5_mutate();
+        derivez_mutate({ payload: { platform: 'derivez' } });
+        dxtrade_mutate({ payload: { platform: 'dxtrade' } });
+    }, [derivez_mutate, dxtrade_mutate, mt5_mutate]);
+
+    /**
+     *
+     * @description This is the modified MT5 accounts that will be used in the CFD account creation.
+     */
+    const modified_mt5_accounts = useMemo(() => {
+        const getAccountInfo = (login?: string) => {
+            return {
+                platform: wallet?.linked_to?.find(linked => linked.loginid === login)?.platform,
```

Comment:
@thisyahlen-deriv Why not using the `combined_cfd_mt5_accounts.platform` here?

Reply:
Because combined_cfd_mt5_accounts is what we use in the store. I want to use the linked_to response as if they add more in the future, we dont need to add it in the store manually.

Reply:
better to rewrite the `combined_cfd_mt5_accounts` somewhere in the constants files to remove dependency from the store